symfony/framework-bundle/3.3/src/Kernel.php

```diff
@@ -34,27 +34,35 @@ public function registerBundles()
         }
     }
 
+    protected function getConfigDir(): string
+    {
+        return dirname(__DIR__).'/config';
+    }
+
+    protected function getVarDir(): string
+    {
+        return dirname(__DIR__).'/var';
+    }
+
     protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader)
     {
         $container->setParameter('container.autowiring.strict_mode', true);
-        $confDir = dirname(__DIR__).'/config';
```

Comment:
I suggest to keep the $confDir variable (same in configureRoutes)

Reply:
done